tokio/src/task/spawn.rs

```diff
@@ -15,6 +15,12 @@ cfg_rt! {
     /// different thread to be executed. The specifics depend on the current
     /// [`Runtime`](crate::runtime::Runtime) configuration.
     ///
+    /// If the task is executed on the current thread it will not begin executing
+    /// until the current task yields to the executor. In other words, `spawn`
```

Comment:
This could be interpreted as "but the multi-threaded runtime might do that!".

Reply:
How about:

> It is guaranteed that `spawn` will not synchronously poll the task being spawned. This means that calling `spawn` while holding a lock does not pose a risk of deadlocking with the spawned task.

Reply:
Updated 👍